### PR TITLE
Add task to publish data to the project data viewer.

### PR DESCRIPTION
**Pull request by MithunKinarullathil**
- [x] Add step to publish data to the project data viewer.
- [x] Add auth token to secrets.
- [x] Add documentation in confluence and README.md. 
---
- [x] Make sure you are opening from a **topic/feature/bugfix branch**
- [x] Ensure that the PR title represents the desired changes
- [x] Ensure that the PR description detail the desired changes
- [x] Link to relevant issues in GitHub or Jira
- [ ] ~~Link to relevant pull requests, esp. upstream and downstream changes~~
- [ ] ~~Ensure you have provided tests - that demonstrates feature works or fixes the issue~~

[^note]:
    Put an `x` into the [ ] to show you have filled the information.
    The template comes from https://github.com/MOV-AI/.github/blob/master/.github/pull_request_template.md
    You can override it by creating .github/pull_request_template.md  in your own repository

### CODE REVIEW

File: .github/workflows/integration-build-product.yml [hunk 1]
```diff
@@ -95,6 +102,7 @@ env:
   SLACK_CHANNEL: ${{ inputs.overwrite_slack_channel }}
   # development slack channel
   #SLACK_CHANNEL: "C05K2KF1UP8"
+  PROJECT_DATA_VIEWER_API: "https://personal-7vf0v2cu.outsystemscloud.com/ProjectDataViewer5/rest/V1//CreateProject"
```
Comment (AlexFernandes-MOVAI) [inline .github/workflows/integration-build-product.yml:105]: should we create a DNS alias for this ?

**Review by AlexFernandes-MOVAI — COMMENTED** [review]
database of https://personal-7vf0v2cu.outsystemscloud.com/ProjectDataViewer5/ is empty

how do you plan to fill it with existing releases ?

**Comment by MithunKinarullathil** [comment]
> database of https://personal-7vf0v2cu.outsystemscloud.com/ProjectDataViewer5/ is empty
> 
> 
> 
> how do you plan to fill it with existing releases ?

I didn't plan on filling the database with existing releases data. It is a question of manual work and how many projects have the apt list file artifact.

File: .github/workflows/integration-build-product.yml (hunk 1 above)
Comment (MithunKinarullathil) [reply .github/workflows/integration-build-product.yml:105]: I'm not knowledgable in this topic. Is it good to do it in this scenario?

Comment (duartecoelhomovai) [reply .github/workflows/integration-build-product.yml:105]: Not a bad idea. Specially because the app is on a temporary in a server.

**Review by duartecoelhomovai — APPROVED** [review]

File: .github/workflows/integration-build-product.yml [hunk 2]
```diff
@@ -918,7 +941,7 @@ jobs:
           path: install_logs/*
 
       - name: Run mobtest
-        if: ${{ needs.Build-Simulator.outputs.skip_simulator == 'false' && inputs.with_simulation == 'true' }}
+        if: ${{ false }}
```
Comment (duartecoelhomovai) [inline .github/workflows/integration-build-product.yml]: Revert it at the end

**Review by duartecoelhomovai — CHANGES_REQUESTED** [review]

File: .github/workflows/integration-build-product.yml (hunk 1 above)
Comment (MithunKinarullathil) [reply .github/workflows/integration-build-product.yml:105]: @AlexFernandes-MOVAI It's ready from my side. Do we have to do the DNS thing?

**Comment by MithunKinarullathil** [comment]
This is ready to go. Last run on project-tugbot: https://github.com/MOV-AI/project-tugbot/actions/runs/6575642089

**Review by duartecoelhomovai — APPROVED** [review]

File: .github/workflows/integration-build-product.yml (hunk 1 above)
Comment (AlexFernandes-MOVAI) [reply .github/workflows/integration-build-product.yml:105]: not blocking

**Review by AlexFernandes-MOVAI — APPROVED** [review]

**Pull request merged**